metaflow/plugins/test_unbounded_foreach_decorator.py

```diff
@@ -97,7 +133,7 @@ def control_task_step_func(self, flow, graph, retry_count):
             task_id = "%s-%d" % (control_task_id.replace("control-", "test-ubf-"), i)
```

Comment:
If this is to remove the need to name the control task something, shouldn't this go away as well?

Reply:
Yes will remove

Reply:
Done